drivers/gpu/drm/bridge/cdns-mhdp.c

```diff
@@ -1565,6 +1565,14 @@ static int cdns_mhdp_sst_enable(struct drm_bridge *bridge)
 
 	bpp = cdns_mhdp_get_bpp(&mhdp->display_fmt);
 
+	if (!cdns_mhdp_bandwidth_ok(mhdp, mode, mhdp->link.num_lanes,
+				    mhdp->link.rate)) {
+		dev_err(mhdp->dev, "%s: Low BW for %s (%u lanes at %u Mbps)\n",
+			__func__, mode->name, mhdp->link.num_lanes,
+			mhdp->link.rate / 100);
+		return -EINVAL;
```

Comment:
I think this is fine, but two comments:
- "Low BW" might not be clear. "Not enough BW", perhaps.
- The error print would be more useful if it would tell the required and available BWs.